### ISSUE TITLE
GPU版本：[W shape_analysis.cpp:841] failed PropagateTensorShapeOnNode with schema

### ISSUE THREAD

**Issue by zhanglv0209** [modelscope/FunASR#2021]
启动： bash run_server.sh   --download-model-dir /workspace/models   --vad-dir damo/speech_fsmn_vad_zh-cn-16k-common-onnx   --model-dir damo/speech_paraformer-large-vad-punc_asr_nat-zh-cn-16k-common-vocab8404-pytorch    --punc-dir damo/punc_ct-transformer_cn-en-common-vocab471067-large-onnx   --lm-dir damo/speech_ngram_lm_zh-cn-ai-wesp-fst   --itn-dir thuduj12/fst_itn_zh   --hotword /workspace/models/hotwords.txt   --certfile 0

报错：
[W shape_analysis.cpp:841] failed PropagateTensorShapeOnNode with schema: 
aten::pad(Tensor self, int[] pad, str mode="constant", float? value=None) -> (Tensor)
%input.887 : Float(*, *, *, requires_grad=0, device=cuda:0) = aten::pad(%input.885, %5190, %5191, %5192), scope: __module.model.encoders/__module.model.encoders.43/__module.model.encoders.43.self_attn/__module.model.encoders.43.self_attn.pad_fn # /usr/local/lib/python3.8/dist-packages/torch/nn/modules/padding.py:23:0

**Comment by dfengpo** [comment]
这是警告而已，可以忽略，也可以在docker内部安装torch解决

**Comment by lyblsgo** [comment]
> 这是警告而已，可以忽略，也可以在docker内部安装torch解决

正解，warning可以忽略
